modules/odc-importing-a-codebase-from-git-to-create-an-application.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// applications/application-life-cycle-management/odc-creating-an-application-using-the-developer-perspective.adoc
+
+[id="odc-importing-a-codebase-from-git-to-create-an-application_{context}"]
+= Importing a codebase from Git to create an application
+
+The following tutorial walks you through the *Import from Git* option to create an application.
+
+Create, build, and deploy an application on {product-title} using an existing codebase in GitHub as follows:
+
+.Prerequisites
+
+* Ensure that you have link:https://docs.openshift.com/container-platform/latest/web-console/web-console.html[logged in to the web console].
+* Ensure that you have switched to the *Developer* perspective.
+//<link:url[label]>
+
+.Procedure
+
+. In the *Add* view, click *From Git* to see the *Import from git* form.
++
+image::odc_import_from_git.png[Import from Git]
+
+. In the *Git* section, enter the Git repository URL for the codebase you want to use to create an application. For this tutorial, enter the URL of this sample nodejs application `https://github.com/sclorg/nodejs-ex`.
+. Optionally, you can click *Show Advanced Git Options*  to add details such as:
+
+* *Git Reference* to point to code in a specific branch, tag, or commit to be used to build the application.
+* *Context Dir* to specify the subdirectory for the application source code, you want to use to build the application.
+* *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
+
++
+For this tutorial you do not need to use the advanced options.
+
+. In the *Builder* section, select the *Node.js*  builder image to see the details of the builder image. If required, you can change the version using the *Builder Image Version* drop-down list.
+. In the *General* section:
+.. In the *Application Name* field, enter `myapp` as the name of the application grouping. Ensure that the application name is unique in a namespace.
+.. The *Name* field to identify the resources created for this application is auto-populated.
++
+NOTE: The resource name must be unique in a namespace. Modify the resource name if you get an error.
+
+. The *Serverless* section is displayed in the *Import from git* form only if the link:https://docs.openshift.com/container-platform/4.1/serverless/installing-openshift-serverless.html#installing-serverless-operator_installing-openshift-serverless[*Serverless Operator*] is installed in your cluster.
+Select *Enable scaling to zero when idle* to create a serverless application that auto scales your Pods to zero to prevent consumption of resources when idle.
++
+NOTE: The *Serverless* option is a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature.
```

Comment:
Notes should be formatted like this: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions